src/transformers/models/swin2sr/modeling_swin2sr.py

```diff
@@ -43,11 +43,11 @@
 
 # General docstring
 _CONFIG_FOR_DOC = "Swin2SRConfig"
-_FEAT_EXTRACTOR_FOR_DOC = "AutoFeatureExtractor"
+_FEAT_EXTRACTOR_FOR_DOC = "AutoImageProcessor"
 
 # Base docstring
-_CHECKPOINT_FOR_DOC = "caidas/swin2sr-classicalsr-x2-64"
-_EXPECTED_OUTPUT_SHAPE = [1, 64, 768]
+_CHECKPOINT_FOR_DOC = "caidas/swin2SR-classical-sr-x2-64"
+_EXPECTED_OUTPUT_SHAPE = [1, 180, 488, 648]
```

Comment:
nit: I remember that we define such variables in some files to avoid the `# Copied from` issue. Here it is only used in a single place below, and the list `[1, 180, 488, 648]` is short enough - we can remove it here and just do

But not a strong comment.

```python
expected_output=_[1, 180, 488, 648]
```